vault/resource_token_auth_backend_role.go

```diff
@@ -44,6 +44,15 @@ func tokenAuthBackendRoleResource() *schema.Resource {
 			DefaultFunc: tokenAuthBackendRoleEmptyStringSet,
 			Description: "List of allowed policies for given role.",
 		},
+		"allowed_policies_glob": {
+			Type:     schema.TypeSet,
+			Optional: true,
+			Elem: &schema.Schema{
+				Type: schema.TypeString,
+			},
+			DefaultFunc: tokenAuthBackendRoleEmptyStringSet,
+			Description: "List of allowed policies with glob match for given role.",
```

Comment:
nit: should probably be documented as a `set` rather than `list`
```suggestion
			Description: "Set of allowed policies with glob match for given role.",
```